functions/index.js

```diff
@@ -103,6 +104,28 @@ exports.handleEvent = functions.firestore
  */
 exports.gerritEndpoint = functions.https.onRequest(gerrit.app)
 
+/**
+ * Handles calls from the Gerrit hook.
+ * @type {HttpsFunction}
+ */
+exports.crowdEndpoint = functions.https.onRequest((req, res) => {
```

Comment:
It looks like we don't do any verification/authentication of the request. I think we should, otherwise, anyone could get added to a Github team by forging the appropriate request.

Reply:
completely agree. I had a TODO to this effect somewhere... must have gotten lost in the rebase. But my plan was to try to implement something like Github webhooks, where the crowd server signs the payload and write an appropriate HTTP header